## TECHIN 513 HW1 – Modifying Signals

Ghea C Suyono

### Assignment 1: Working with Sound Files

#### Part A

In [2]:
# Import relevant libraries

import numpy as np
import matplotlib.pyplot as plt
from scipy.io import wavfile as wav
import simpleaudio as sa

##### Use Scipy.io to read audio file
Note: 
- fs1 = sampling frequency 1
- x1 = audio vector


In [13]:
#import train32 file
fs1, x1 = wav.read('/workspaces/513_HW1/sounds/train32.wav')

print('Sampling rate:', fs1, 'Audio vector', x1)
x1.shape



Sampling rate: 32000 Audio vector [-901 -943 -646 ...  330  169   45]


(50313,)

The data above shows that:
1. The sampling rate of train32 file is 32kHz 
2. The shape of audio data array is 1D array (mono) with 50,313 samples.